src/openlcb/MemoryConfigClient.hxx

```diff
@@ -135,9 +220,19 @@ public:
     {
     }
 
+    enum ResultCodes
+    {
+        // Internal error codes generated by the send flow
+        OPERATION_PENDING =
+            DatagramClient::OPERATION_PENDING, //< cleared when done is called.
```

Comment:
These doxygen comments need another '/' to be valid.

Reply:
done